zbx_clickhouse_monitor.sh

```diff
@@ -27,6 +27,9 @@ if [ -z "$ITEM" ]; then
 	exit 1
 fi
 
+# Collect for additional parameters if available. Always get last argument if args count > 2.
+if [ $# -gt 2 ] ; then  ADD_FLAGS="${@: -1}" ; else ADD_FLAGS="" ; fi
```

Comment:
What if we have, say, 4 params? Does it mean that 3-rd param would be lost?

Reply:
May be we'd like to accumulate not the last param, but all params, starting from 3-rd and up to the end?

Reply:
So this is really discussible.
Original script doesn't handle more than 2 args.
1) yes, currently 3rd param will be lost. I think about it the same way as about *args in python, which should be always last. But it's just a thought.
2) Collecting args is a bad idea. It makes impossible to rely on positional arguments without incrementing this on each enhancement. But this will break backward compatibility.

Reply:
Ok, let it be for now, we may return to this in future, if need be.